### ISSUE TITLE
Tab completion not working in REPL

### ISSUE THREAD

**Issue by sloede** [analytech-solutions/CBinding.jl#77]
Using your fork https://github.com/krrutkow/P4est.jl/tree/update-to-cbinding-v1 I tried to use P4est.jl on the REPL. Even though I faintly recall it used to work at some point in the past, tab-completion in the REPL is not working anymore, at least for me:
```julia
julia> using P4est

julia> conn_ptr = p4est_connectivity_new_periodic()
CBinding.Cptr{var"c\"struct p4est_connectivity\""}(0x000000000441d5e0)

julia> propertynames(conn_ptr)
(:num_vertices, :num_trees, :num_corners, :vertices, :tree_to_vertex, :tree_attr_bytes, :tree_to_attr, :tree_to_tree, :tree_to_face, :tree_to_corner, :ctt_offset, :corner_to_tree, :corner_to_corner)
```
Thus although `propertynames` seems to return the correct list of properties, when I enter `conn_ptr.` and hit `Tab`, it just gives me a list of hundreds of symbols. Is this a known issue?

**Comment by krrutkow** [comment]
This worked in pre-v1 code because `Caccessor` was a struct type wrapping a pointer.  This no longer works due to https://github.com/JuliaLang/julia/issues/41652 and the fact that pointers (non-structs) are used directly now.  There is at least one slightly risky workaround that fixes this anyways.

**Comment by sloede** [comment]
OK, thanks for letting me know. I'll keep tabs on JuliaLang/julia#41652 to see what happens.

**Comment by krrutkow** [comment]
A fix for JuliaLang/julia#41652 has been merged into Julia.
